app/kafka/manager/KafkaManagerActor.scala

```diff
@@ -172,6 +187,8 @@ object ClusterConfig {
             curatorConfig,
             enabled,version,
             jmxEnabled.getOrElse(false),
+            jmxUser.getOrElse(Some("")),
```

Comment:
just use jmxUser and jmxPass directly, no need to generate default value here since they are Option[String] already.